go.mod

```diff
@@ -24,7 +24,7 @@ require (
 	github.com/onsi/gomega v1.15.0
 	github.com/openshift/api v0.0.0-20200331152225-585af27e34fd
 	github.com/openshift/client-go v0.0.0-20200326155132-2a6cd50aedd0
-	github.com/operator-framework/api v0.11.0
+	github.com/operator-framework/api v0.11.1
```

Comment:
Just for other reviewers: https://github.com/operator-framework/api/commit/160240770853d11a62b3ff721b3ffed2d8edf06d is tagged v0.11.1 on the o-f/api side.